index.html

```diff
@@ -0,0 +1,8 @@
+<html>
+  <head>
+    <meta http-equiv="Refresh" content="0; url=https://localhost:8443/test/test-pages/">
+  </head>
+  <body>
+    Redirecting...
+  </body>
+</html>
```

Comment:
This is a simple helper file that redirects to the test-page route when you hit the root.